src/components/Book.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import './Book.css';
+import { removeBook } from '../redux/Books/books';
+
+const Book = ({ book }) => {
+  const dispatch = useDispatch();
+  const { id } = book;
+  const percentage = 66;
+
+  return (
+
+    <div className="bookContents">
+      <div className="content1">
+        <div className="content1FirstChild">
+          <p className="genre">{book.category}</p>
+          <h2 className="title">{book.title}</h2>
+          <p className="author">{book.author}</p>
+        </div>
```

Comment:
- [ ] please make sure to align the content as in the Design in order to match it
- Yours 
![image](https://user-images.githubusercontent.com/61790587/178991037-2da59d5f-7c9f-4435-93c4-872bd0c40c0f.png)
- Design 
![image](https://user-images.githubusercontent.com/61790587/178991056-3a590823-566c-4652-bfa8-b50fb6186a18.png)